cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -150,6 +150,28 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
```

Comment:
Why is this marked for rhel_testing? Doesn't seem appropriate given the test.

Reply:
I see, other test functions in the module are decorated similarly. I guess this does, to a degree, test the 'backend' rhel capabilities of the CFME host.